Feature/fix nfs provisioner version (#12)
3scale/multikloud: README.md

* k3d - e.g. `brew install k3d`
* ansible (for generating the k8s templates)

### RWX PVC support

* 3scale System component needs a RWC PVC for `system-storage`:
  * Configuration files read by the System component at run-time
  * Static files (HTML, CSS, JS, etc) uploaded to System by its CMS feature, for the purpose of creating a Developer Portal
  * Note that System can be scaled horizontally with multiple pods uploading and reading said static files, hence the need for a RWX PersistentVolume

> * For dev purpose, it can deployed on the k8s cluster a small [nfs-provisioner](kubernetes/nfs/) in order to create a RWX `PersistentVolumeClaim` without using S3 or any external NFS...

Reviewer: ```suggestion
* For dev purposes, a small [nfs-provisioner](kubernetes/nfs/) can be deployed in the k8s cluster in order to create a RWX `PersistentVolumeClaim` without using S3 or any external NFS...
```
